packages/fetch/src/vfetch.ts

```diff
@@ -0,0 +1,18 @@
+import { ClientException } from '@voiceflow/exception';
+
+export type FetchAPI<Request, RequestInit, Response> = (input: string | URL | Request, init?: RequestInit) => Promise<Response>;
+
+export const vfetch: {
```

Comment:
I'm not sold on this, I would almost rather opt-in to catching and error and wrapping it rather than needing to use this. But I do see the benefit.

To @jonahsnider 's point earlier about wanting to use a higher abstraction over fetch, axios has an [interceptor](https://www.npmjs.com/package/axios#interceptors) that could be used to wrap errors. It would still require using this lib, but just wanted to point that out.